### ISSUE TITLE
pack build fails to pull image when docker is configured to use gcr credential helper 

### ISSUE THREAD

**Issue by jldec** [buildpack/pack#74]
Using own build of pack cli from master branch
(git sha: 1974cedf4fc4836b1df2c30b0cbcbce5dffbd2d3)

I can successfully build using builder from public repo on dockerhub
e.g. `pack build jldec/java-hello:v1 --builder projectriff/builder --path .`

But substituting for a builder which requires a gcr credential helper fails, even though `docker pull` of the same image succeeds.

```sh
$ echo $PFS_BUILDER_IMAGE
gcr.io/cf-spring-pfs-release/projectriff-builder-a527160a809f5f8d339af74b304e6c46:0.0.1-snapshot-ci-31a4547a8f07

$ pack build jldec/java-hello:vp1 --builder $PFS_BUILDER_IMAGE --path . 
2018/11/25 17:33:50 Using user provided builder image 'gcr.io/cf-spring-pfs-release/projectriff-builder-a527160a809f5f8d339af74b304e6c46:0.0.1-snapshot-ci-31a4547a8f07'
2018/11/25 17:33:50 Pulling builder image 'gcr.io/cf-spring-pfs-release/projectriff-builder-a527160a809f5f8d339af74b304e6c46:0.0.1-snapshot-ci-31a4547a8f07' (use --no-pull flag to skip this step)
Error: Error response from daemon: unauthorized: You don't have the needed permissions to perform this operation, and you may have invalid credentials. To authenticate your request, follow the steps in: https://cloud.google.com/container-registry/docs/advanced-authentication

$ docker pull $PFS_BUILDER_IMAGE
0.0.1-snapshot-ci-31a4547a8f07: Pulling from cf-spring-pfs-release/projectriff-builder-a527160a809f5f8d339af74b304e6c46
Digest: sha256:03082ad86323f70a699c784ce8dc5afd9bc9554ba500f126bd7d8daee09a8eb5
Status: Image is up to date for gcr.io/cf-spring-pfs-release/projectriff-builder-a527160a809f5f8d339af74b304e6c46:0.0.1-snapshot-ci-31a4547a8f07
```

NOTE: I'm using the "standalone" gcr credential helper described [here](https://cloud.google.com/container-registry/docs/advanced-authentication).

```sh
gcloud components install docker-credential-gcr
docker-credential-gcr configure-docker
```

cc @scothis

**Comment by ekcasey** [comment]
reopening until we release `pack`@`v0.0.9` with the fix

**Comment by ekcasey** [comment]
released in `pack@v0.0.9`
